charts/aws-ingress/values.yaml

```diff
@@ -44,3 +45,21 @@ ingress:
     selector:
       key: app
       value: minio # (currently) fake-aws-s3 chart uses 'minio', minio-external chart uses 'minio-external'
+  teamSettings:
+    https:
+      externalPort: 443
+      sslCert: arn:aws:iam::00000-accountnumber-00000:server-certificate/example.com
+      sslPolicy: ELBSecurityPolicy-TLS-1-2-2017-01
```

Comment:
The security policy isn't as good as it can be, let's use a custom one that is a bit more secure as on our own production, please.

Reply:
I don't think this is possible using the k8s API though, at least not when looking at: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/legacy-cloud-providers/aws/aws.go

I.e., I don't see a way to specify custom protocols (i.e., custom policy attributes). Happy to be proved wrong 😬

Reply:
You're right that custom policy attributes seem no longer supported by AWS on the new types of load balancers at all anymore. You could use `ELBSecurityPolicy-FS-1-2-Res-2019-08` though, which is the best AWS currently offers. Search for `ELBSecurityPolicy-FS` on [this page](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/create-https-listener.html#describe-ssl-policies)